autorgb/ProcessImages.py

```diff
@@ -119,11 +183,62 @@ def update_progress(self):
 
     # Open an Image
     def open_image(self, path):
+        """
+        Open an image file from a specified file path.
+
+        Args:
+            path (str): The file path of the image to be opened, including the 
+            filename.
+
+        Returns:
+            PIL.Image.Image: The opened image object.
+
+        Raises:
+            PIL.UnidentifiedImageError: If the image cannot be identified and 
+            opened.
+            FileNotFoundError: If the specified file does not exist.
+            PermissionError: If read permission is denied for the specified 
+            file path.
+
+        Example:
+            >>> path = "path/to/open/image.png"
+            >>> opened_image = open_image(path)
+
+        Notes:
+            - The function uses PIL to open the image.
+        """
         newImage = Image.open(path)
         return newImage
 
     # Save Image
     def save_image(self, image, path):
+        """
```

Comment:
Given the simplicity of this method, this docblock can probably be a little less verbose. Let's drop the raises, example, and notes section of the docblock.

Reply:
Let's actually drop the raises, example and notes sections on all of these, as they're not really needed in this context. They would be more useful in a general purpose library, I think. The main thing I want these docstrings to communicate is a general sense of how the method works.